src/google/mod.rs

```diff
@@ -3,8 +3,12 @@
 mod constants;
 
 mod credential;
-pub use credential::Credential as GoogleCredential;
+pub use credential::CredentialAccount as GoogleAccount;
 pub use credential::CredentialLoader as GoogleCredentialLoader;
+pub use credential::ExternalAccount as GoogleExternalAccount;
+pub use credential::ServiceAccount as GoogleCredential;
```

Comment:
I didn't want to break anything, but I'd consider making this `#[deprecated]` in favor of `GoogleServiceAccount`

Reply:
I don't want to expose the internal details to users. From my point of view, both `ExternalAccount` and `ServiceAccount` should not be exposed to users.

Reply:
We still need to change the export?